build.bat

```diff
@@ -165,8 +168,9 @@ if %build_deps% == 1 (
   echo building opencc.
   pushd deps\opencc
   cmake . -B%build_dir% %deps_cmake_flags%^
-  -DBUILD_SHARED_LIBS=OFF^
-  -DBUILD_TESTING=OFF
+  -DCMAKE_SKIP_BUILD_RPATH=OFF^
+  -DBUILD_TESTING=OFF^
+  -DUSE_SYSTEM_MARISA=ON

```

Comment:
msvc build failed:

    D:\a\librime\librime\deps\opencc\src\MarisaDict.cpp(23): fatal error C1083: Cannot open include file: 'marisa.h': No such file or directory

Reply:
That is wired. The CI tests of my fork are succeed.